pkg/cmd/admin/groups/new.go

```diff
@@ -49,10 +52,18 @@ func NewCmdNewGroup(name, fullName string, f *clientcmd.Factory, out io.Writer)
 				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
 			}
 
+			kcmdutil.CheckErr((kcmdutil.ValidateOutputArgs(cmd)))
+
 			kcmdutil.CheckErr(options.AddGroup())
+
+			mapper, _ := f.Object(false)
+			shortOutput := kcmdutil.GetFlagString(cmd, "output") == "name"
+			kcmdutil.PrintSuccess(mapper, shortOutput, out, "group", options.Group, "created")
```

Comment:
I think this output should be generated in the `AddGroup` method using a `PrinterFunc` and the `Out` provided by command (or a discard for unit tests, or the glog-ing out that @stevekuznetsov made a while back).